src/mlpack/bindings/julia/CMakeLists.txt

```diff
@@ -0,0 +1,158 @@
+if (BUILD_JULIA_BINDINGS)
+  ## We need to check here if Julia is even available.  Although actually
+  ## technically, I'm not sure if we even need to know!  For the tests though we
+  ## do.  So it's probably a good idea to check.
+  find_package(Julia 0.7.0)
+  if (NOT JULIA_FOUND)
```

Comment:
Should we print out a warning or something like that, since the user ask to build the julia bindings?

Reply:
Ah, good point, fixed in dda0fc9a.  @Yashwants19 and I were talking about how to do that for Go bindings in #1884.  @Yashwants19: take a look at dda0fc9a and you could adapt the same strategy for the Go bindings.

Basically this means that if the user did not specify `-DBUILD_JULIA_BINDINGS=ON` and Julia is not found, then configuration will complete successfully but Julia bindings will not be built.  If the user *does* specify `-DBUILD_JULIA_BINDINGS=ON` and Julia is *not* found, then a fatal error will be issued during configuration.